mage/vars.go

```diff
@@ -291,7 +291,7 @@ func buildFlags() []string {
 }
 
 func importEnvFromMakefile() (env map[string]string) {
-	cmd := exec.Command("make", "-f", "Makefile.buildx", "magnet-vars")
+	cmd := exec.Command("make", "--quiet", "-f", "Makefile.buildx", "magnet-vars")
```

Comment:
Opportunistic improvement to avoid spam during build:
```
2021/11/11 15:38:57 Skip line that does not look like magnet envar: "make[1]: Entering directory '/path/to/go/src/github.com/gravitational/gravity'"
2021/11/11 15:38:57 Skip line that does not look like magnet envar: "make[1]: Leaving directory '/path/to/go/src/github.com/gravitational/gravity'"

```